json-rpc/types/src/request.rs

```diff
@@ -713,6 +728,64 @@ mod test {
         serde_json::from_value::<GetStateProofParams>(value).unwrap();
     }
 
+    #[test]
+    fn get_accumulator_consistency_proof() {
+        // Array with all params
+        let value = serde_json::json!([11, 42]);
+        serde_json::from_value::<GetAccumulatorConsistencyProofParams>(value).unwrap();
+
+        // Array with too many params
+        let value = serde_json::json!([11, 42, 7]);
+        serde_json::from_value::<GetAccumulatorConsistencyProofParams>(value).unwrap_err();
```

Comment:
Btw you could probably collapse a lot of this by making 2 vecs, 1 are successes, and the other are failures to iterate over.  Then just comment on the rows if you want.

Reply:
If done naively that can end up making debugging errors more difficult since you then have a more difficult time seeing what the input is.